### PR TITLE
Switch to pytest

### PR DESCRIPTION
**Pull request by xoviat**

### CODE REVIEW

**Comment by xoviat** [comment]
@matthew-brett Is Travis-CI enabled?

**Comment by matthew-brett** [comment]
Yes, its enabled - I'm not sure why it's not triggering here.

File: .travis.yml [hunk 1]
```diff
@@ -33,15 +33,15 @@ install:
...
   - source multibuild/osx_utils.sh
   - get_macpython_environment $VERSION venv
-  - pip install wheel nose
+  - pip install wheel pytest
   - if [ -n "$BUILD_WHEELS" ]; then
         pip install cython;
         (cd wheel_makers && ./make_wheels.sh);
     fi
-  - pip install .
+  - pip install -e .
```
Comment (matthew-brett) [inline .travis.yml]: This was deliberate (to test the installed distribution).

File: .travis.yml [hunk 2]
```diff
@@ -33,15 +33,15 @@ install:
...
 
 script:
   - echo $PWD
   - mkdir tmp_test_dir
   - cd tmp_test_dir
-  - nosetests delocate
+  - pytest delocate
```
Comment (matthew-brett) [inline .travis.yml]: To test the installed distribution `pytest --pyargs delocate`

**Comment by matthew-brett** [comment]
Sorry - one more mention of nose in `doc/devel/make_release.rst`

**Comment by xoviat** [comment]
@matthew-brett Done; no need to wait for the CI.

**Comment by matthew-brett** [comment]
Thanks for those.

**Pull request merged**